spring-cloud-stream-app-generator/pom.xml

```diff
@@ -154,6 +154,25 @@
 						</syslog-source>
 						<task-launcher-local-sink />
 						<task-launcher-yarn-sink />
+						<task-launcher-cloudfoundry-sink>
+							<forceDependencies>
+								<dependency>
+									<groupId>io.projectreactor</groupId>
+									<artifactId>reactor-core</artifactId>
+									<version>3.0.2.RELEASE</version>
```

Comment:
We should introduce properties for those and keep them close to the others, or we're guaranteed to forget upgrading them :(